frontend/lib/justfix-navbar.tsx

```diff
@@ -53,11 +49,28 @@ const JustfixMenuItems: React.FC<{}> = () => {
   );
 };
 
+const JustfixUserMenuItems: React.FC<{}> = () => {
+  return (
+    <>
+      <Link
+        className="navbar-item"
+        to={JustfixRoutes.locale.accountSettings.home}
+      >
+        Account settings
```

Comment:
I didn't bother internationalizing this for now because we don't support user account creation/login for app.justfix.nyc.